onnx/defs/schema.cc

```diff
@@ -622,6 +622,12 @@ void OpSchema::ParseAndSetTypes(
   }
 }
 
+OpSchema& OpSchema::FunctionBody(const FunctionProto& func_proto) {
```

Comment:
I think we need to specify only a list of nodes here. FunctionProto has extra fields (name, inputs, outputs, attributes) which replicate what is separately specified in an OpSchema. This is an unnecessary duplication (what happens if different values are specified in the two places? There is no check for that.)

Reply:
I checked that in verification.. but its definitely right we need to remove this. Ill change it.